charts/dapr/crds/subscription.yaml

```diff
@@ -2,6 +2,8 @@ apiVersion: apiextensions.k8s.io/v1
 kind: CustomResourceDefinition
 metadata:
   name: subscriptions.dapr.io
+  labels:
+    app.kubernetes.io/part-of: "dapr"
```

Comment:
@shubham1172 Can this be read here and above places from below values.yaml file ? Is there any benefit of not doing it ?

Reply:
We are using method 1 from https://helm.sh/docs/chart_best_practices/custom_resource_definitions/
<img width="1202" alt="image" src="https://user-images.githubusercontent.com/22132944/192232540-4b434ec2-89c7-4459-91ab-0b469b9747fc.png">

So templating is unsupported!